docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -159,18 +159,29 @@ start the {dfeed} for the change to be applied.
 == 7. Set the model memory limit
 
 The `model_memory_limit` job configuration option sets the approximate maximum 
-amount of memory resources required for analytical processing. If this variable 
-is set too low for the job and the limit is approached, data pruning becomes 
-more aggressive. Upon exceeding this limit, new entities are not modeled.
-
-Use model memory estimation to have a better picture of the memory needs of the 
-model. Model memory estimation happens automatically when you create the job in 
-{kib} or you can call the 
-{ref}/ml-estimate-model-memory.html[Estimate {anomaly-jobs} model memory API] 
-manually. The estimation is based on the analysis configuration details for the 
-job and cardinality estimates for the fields it references. You can update the 
-memory settings of an existing job, but the job must be closed.
-
+amount of memory resources required for analytical processing. When you create
+an {anomaly-job} in {kib}, it provides an estimate for this limit. The estimate 
+is based on the analysis configuration details for the job and cardinality 
+estimates for the fields it references. You can also obtain this information
+from the
+{ref}/ml-estimate-model-memory.html[estimate {anomaly-jobs} model memory API].  
```

Comment:
It's worth adding that the cardinality estimates used by Kibana come from running aggregations against the source indices of the datafeed at the time Kibana is being used to create the job.  If these are not reflective of the cardinality the job will experience while running (for example because ingest hasn't yet started at the time the job is created) then the memory estimate will be wrong.

If you use the estimate anomaly detection jobs model memory API then you must provide your own cardinality estimates.

Reply:
Thanks, I've interwoven those details in these paragraphs.